### ISSUE TITLE
[BUG] cimage shows up every text message

### ISSUE THREAD

**Issue by brunocalado** [p4535992/foundryvtt-chat-media#1]
Hi. Every text message on the chat "cimage" appears. I just typed the message and pressed enter

![image](https://github.com/p4535992/foundryvtt-chat-media/assets/4999783/e524bc3d-ffc5-4016-8a89-cdf30e7d4ff0)

fvtt 11307
current chat media build
google chrome
windows 10

**Comment by tehguitarist** [comment]
Unfortunately this is still happening for me on 3.3.5

**Comment by p4535992** [comment]
@tehguitarist tested and retested is working, did you have any other module active ? what system are you using ? what version of foudnry are you using ?

**Comment by tehguitarist** [comment]
Hey, sorry, was during a game. I just tested on a clean world and it didn't happen, I'll do some digging to see if I can find a module that's causing it. Strangely, I opened a world in Firefox, and it didn't happen. But then opening the same world in Edge (different cookies - so different settings) and it started happening again.

The only thing I can think of is that I had to manually enter the manifest again to update, so maybe it was some weird cacheing thing (I connect via a cloudflare proxy). So sorry for the bother!
